packages/block-editor/src/components/rich-text/index.js

```diff
@@ -491,6 +496,7 @@ export function RichTextWrapper(
 						inputEvents,
 					} ),
 					anchorRef,
+					anchorCallbackRef,
```

Comment:
I think it's okay to pass `setAnchorElement` here directly without the need to wrap in `useCallback`. As long as the ref callback has a stable reference (the state setter does), it won't be called on every render.

We use similar patterns elsewhere for Popover positioning.

https://github.com/WordPress/gutenberg/blob/feeceda6431848cb1d8a1ea00c906d38eced2248/packages/block-editor/src/components/url-popover/image-url-input-ui.js#L59-L61